### DISCUSSION TITLE
Issue while running SU2 code with Intel OneAPI compiler

### DISCUSSION THREAD

**Discussion by shunido** [su2code/SU2#2202] [Q&A]
Hi
Let me first introduce the environment , so If anybody got a similar one better to assist

We have a small cluster of 16 nodes with 2 AMD Milan (AMD EPYC 7543) CPUs (32 cores each) 

a headnode for management , jobs to be submitted using slurm and network for communication is using Mellanox infiniband 

compiling the SU2 code using the Intel OneApi can be done without issues 

But later when trying to submit job , it will stuck , it will not produce any output 

If we compile another SU2 with OpenMPI , then this runs fine without any issues 

If we use the online precompiled SU2 , also it works without any issues , I think that is compiled with MPICH

Does any one knows this please or any suggestions for compiling with Intel ? 

Thank you

**Comment by mehulbakhshi02** [comment]
Hello
I am facing exactly the same issue with OneApi. Kindly let me know you are able to find any solution for this.

Thank you
